File: frontend/lib/models/mechanic_club_link_request_dto.dart
class MechanicClubLinkRequestDto {
  final int? clubId;
  final bool attach;

  const MechanicClubLinkRequestDto({
    this.clubId,
    required this.attach,
  });

  Map<String, dynamic> toJson() => {
        'clubId': clubId,
        'attach': attach,
      };
}

File: backend/CoreService/src/main/java/ru/bowling/bowlingapp/DTO/MechanicClubLinkRequestDTO.java
package ru.bowling.bowlingapp.DTO;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MechanicClubLinkRequestDTO {
    private Long clubId;
    private boolean attach;
}
